File: fibo.rb
def fibo(n)
    case n
    when 0
        return 0
    when 1
        return 1
    else
        return fibo(n-1) + fibo(n-2)
    end
end

i = 0
until i == 20
    puts fibo(i)
    i += 1
end

File: fibo.py
def fibo(n):
    if (n == 0):
        return 0
    elif (n == 1):
        return 1
    else:
        return fibo(n-1) + fibo(n-2);

i = 0

while i < 20:
    print fibo(i)
    i += 1
